docs/getting-started/advanced-setting.asciidoc

```diff
@@ -175,3 +175,52 @@ By default, Elastic prebuilt rules in the *Rules* and *Rule Monitoring* tables i
 == Manage alert tag options
 
 The `securitySolution:alertTags` field determines which options display in the alert tag menu. The default alert tag options are `Duplicate`, `False Positive`, and `Further investigation required`. You can update the alert tag menu by editing these options or adding more. To learn more about using alert tags, refer to <<apply-alert-tags>>.
+
+[discrete]
+[[exclude-cold-frozen-data-rule-executions]]
+== Exclude cold and frozen data from rule executions 
+
+To ensure rules don't search cold and frozen data when executing, specify cold and frozen {ref}/data-tiers.html[data tiers] in the `excludedDataTiersForRuleExecution` field. Multiple data tiers must be separated by commas, for example: `data_frozen`, `data_cold`. This setting is turned off by default; turning it on can improve rule performance and reduce execution time. 
+
+The `excludedDataTiersForRuleExecution` setting does not apply to {esql} or {ml} rules. 
+
+[TIP]
+====
+
+The `excludedDataTiersForRuleExecution` setting applies to all rules in a {kib} space. To only exclude cold and frozen data from specific rules, add a {ref}/sql-rest-filtering.html[Query DSL] filter to the rules you want affected. Add the filter when creating a new rule or updating an existing one. 
```

Comment:
Also, the Query DSL link feels a little distracting since it jumps to Elasticsearch API docs — now the reader has to mentally translate raw Elasticsearch API stuff back into the context of Elastic Security and how to add a filter to a detection rule. I think the example queries below convey most of what we need.

Alternatively, linking to https://www.elastic.co/guide/en/elasticsearch/reference/master/query-dsl.html could be somewhat more relevant to our users, since it looks like a more general reference on the Query DSL language, not its API.

Reply:
I don't think we need to reference the DSL docs. The sample filters below just need slight modifications to use (users really just need to update the names of their cold/frozen data tiers), so more information on DSL might not be needed. I'll update your comment to remove the ref.